graphics/libopenraw-legacy/Portfile

```diff
@@ -0,0 +1,96 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           boost 1.0
+PortGroup           compiler_blacklist_versions 1.0
+
+name                libopenraw-legacy
+version             0.3.4
+revision            0
+
+categories          graphics
+license             LGPL-3+
+maintainers         {mascguy @mascguy} {@barracuda156 gmail.com:vital.had} openmaintainer
+
+description         libopenraw is an ongoing project to provide a free software implementation \
+                    for decoding camera RAW files.
+long_description    ${description} \
+                    One of the main reasons is that dcraw is not suited for easy integration into \
+                    applications, and there is a need for an easy to use API to build free software \
+                    digital image processing applications. It also has the goal to address features \
+                    missing from dcraw such as meta-data decoding and easy thumbnail extraction.
+
+homepage            https://libopenraw.freedesktop.org/${name}/
+master_sites        https://libopenraw.freedesktop.org/download/
+
+use_xz              yes
+
+distname            libopenraw-${version}
+checksums           ${distname}${extract.suffix} \
+                    rmd160  2372f9eaa4b4f806a0533c867cb7656186a47d59 \
+                    sha256  c365210796161e017ce87d09bd746237d845ade9ab74faaafcede89382b95640 \
+                    size    570500
+
+conflicts           libopenraw
+
+# https://gitlab.freedesktop.org/libopenraw/libopenraw/-/issues/13
+patchfiles          patch-Makefile.am.diff \
+                    patch-ifdentry.hpp.diff \
+                    patch-Make-CR3-support-optional.diff
+
+depends_build       port:pkgconfig \
+                    port:autoconf-archive \
+                    port:cctools \
+                    port:grep \
+                    port:gsed \
+                    port:gawk
+
+depends_lib         port:libxml2 \
+                    port:curl \
+                    path:include/turbojpeg.h:libjpeg-turbo
+
```

Comment:
the boost PG adds dependencies so you should probably use "-append" when declaring other dependencies.

Reply:
Thank you, will do it. While it seems to work fine without -append, adding it is a safe bet.

Reply:
UPD. Done and rebased to master.